locomotion-sdk/src/LocomotionRouter.js

```diff
@@ -2,10 +2,10 @@ import React, { useEffect } from 'react';
 import Config from 'react-native-config';
 import { initStripe } from '@stripe/stripe-react-native';
 import 'react-native-gesture-handler';
-
+import { NavigationContainer } from '@react-navigation/native';
```

Comment:
use `MainProvider` instead of `LocomotionRouter` for more providers, keep this clean.

Reply:
it creates a loop of imports, wasn't sure a better way to solve that

Reply:
fix this loop by moving `StateContext` / `MainProvider` to its own folder